packages/editor-ui/src/composables/usePushConnection.ts

```diff
@@ -230,11 +230,33 @@ export function usePushConnection({ router }: { router: ReturnType<typeof useRou
 				return false;
 			}
 
+			let showedSuccessToast = false;
```

Comment:
I know this is not ideal, but it minimizes the extent of changes to make. `pushMessageReceived` is in dire need of refactoring.